test/e2e/DevServer.test.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const testBin = require('../helpers/test-bin');
+const isWebpack5 = require('../helpers/isWebpack5');
+
+describe('DevServer', () => {
+  const webpack5Test = isWebpack5 ? it : it.skip;
```

Comment:
We should not skip some tests for webpack v4, because we still support it

Reply:
Only skipping for webpack-5 specific tests. for ex when `target: [web, web-worker]` in config.